src/tools/wasm-split/wasm-split.cpp

```diff
@@ -203,12 +201,24 @@ void writeSymbolMap(Module& wasm, std::string filename) {
   runner.run();
 }
 
-void writePlaceholderMap(const std::map<size_t, Name> placeholderMap,
-                         std::string filename) {
+void writePlaceholderMap(
+  Module& wasm,
+  const std::unordered_map<Name, std::map<size_t, Name>> placeholderMap,
```

Comment:
Based on the call sites, it looks like this should be a reference (even though it wasn't before).